### PR TITLE
Mar jose123 filament webhook server

### PR DESCRIPTION
**Pull request by MarJose123**

### CODE REVIEW

**Review by saade — CHANGES_REQUESTED** [review]
Please do not include the word "Filament" in the name or slug of your plugin, or the file names. It is redundant, as all plugins on the website are for Filament.

File: content/plugins/marjose123-webhook-server.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Webhook Server
+slug: filament-webhook-server
```
Comment (saade) [inline content/plugins/marjose123-webhook-server.md:3]:
```suggestion
slug: marjose123-webhook-server
```

File: content/plugins/marjose123-webhook-server.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+author_slug: marjose123
+categories: [developer-tool]
+description: Send webhooks from Filament apps.
+discord_url: https://discord.com/channels/883083792112300104/1069185993770536990
+docs_url: https://raw.githubusercontent.com/MarJose123/filament-webhook-server/2.x/README.md
+github_repository: https://github.com/MarJose123/filament-webhook-server
+has_dark_theme: true
+has_translations: true
+image: filament-webhook-server.jpg
```
Comment (saade) [inline content/plugins/marjose123-webhook-server.md:12]:
```suggestion
image: marjose123-webhook-server.jpg
```

also rename your image to match the suggestion above

**Review by saade — CHANGES_REQUESTED** [review]

**Comment by saade** [comment]
@MarJose123 your plugin's image file name needs renaming too

**Comment by MarJose123** [comment]
Hi @saade my apologies, I've updated now the image filename for your review.
:clinking_glasses:

File: content/plugins/marjose123-webhook-server.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Webhook Server
+slug: marjose123-webhook-server
+author_slug: marjose123
+categories: [developer-tool]
+description: Send webhooks from Filament apps.
+discord_url: https://discord.com/channels/883083792112300104/1069185993770536990
+docs_url: https://raw.githubusercontent.com/MarJose123/filament-webhook-server/2.x/README.md
+github_repository: https://github.com/MarJose123/filament-webhook-server
```
Comment (awcodes) [inline content/plugins/marjose123-webhook-server.md:9]: Please remove the github domain. Should just be 'MarJose123/filament-webhook-server'

Comment (MarJose123) [reply content/plugins/marjose123-webhook-server.md]: resolved

**Review by awcodes — APPROVED** [review]

**Comment by danharrin** [comment]
Hey! Could you please ensure that image URLs in your plugin's `README` file are absolute URLs rather than relative links? Then we can embed them on our website correctly. Thanks!

**Pull request merged**